cmd/api/routes.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"github.com/go-chi/chi"
+	"github.com/go-chi/chi/v5/middleware"
+	"github.com/go-chi/cors"
+	"github.com/vladyslavpavlenko/genesis-api-project/internal/handlers"
+	"net/http"
+)
+
+func routes() http.Handler {
+	mux := chi.NewRouter()
+	mux.Use(middleware.Logger)
+
+	mux.Use(cors.Handler(cors.Options{
+		AllowedOrigins:   []string{"https://*", "http://*"},
+		AllowedMethods:   []string{"GET", "POST", "PUT", "DELETE", "OPTIONS"},
+		AllowedHeaders:   []string{"Accept", "Authorization", "Content-Type", "X-CSRF-Token"},
```

Comment:
u have csrg token here?

Reply:
The "X-CSRF-Token" header wasn't necessary, I agree with you. I added it a while ago just in case I ever need it. I'll remove it for now.